logstash-core/locales/en.yml

```
@@ -139,9 +139,11 @@ en:
       configtest-flag-information: |-
         You may be interested in the '--configtest' flag which you can use to validate
         logstash's configuration before you choose to restart a running system.
+      tags-illegal-deprecated: >-
+        The flag '--event_api.tags.illegal' is deprecated and will be removed in version 9.
       tags-illegal-warning: >-
         Setting `event_api.tags.illegal` to `warn` allows illegal values in the reserved `tags` field, which may crash pipeline unexpectedly.
-        This flag value is deprecated and may be removed in a future release.
+        This flag value is deprecated and will be removed in version 9.
```

We are deprecating `event_api.tags.illegal` setting, maybe removing value (value is `warn` here) would make more sense.
```suggestion
        This flag is deprecated and will be removed in version 9.
```